homeassistant/components/danfoss_air/binary_sensor.py

```diff
@@ -0,0 +1,56 @@
+"""
+Support for the for Danfoss Air HRV binary sensor platform.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.danfoss_air/
+"""
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.components.danfoss_air import DOMAIN \
+     as DANFOSS_AIR_DOMAIN
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
```

Comment:
Rename `add_devices` to `add_entities`.

Reply:
Agree that add_entities is a more logical name. Will fix in a couple of days.